test/system.test.ts

```diff
@@ -82,6 +82,11 @@ describe('System.', () => {
         it('Fulfilled', () => fin.System.getProxySettings().then(() => assert(true)));
     });
 
+    describe('getHostSpecs()', () => {
+
+        it('Fulfilled', () => fin.System.getHostSpecs().then(specs => assert(true)));
```

Comment:
• `specs` here is unused. Can simply forgo it: `.then(() => assert(true)));`
• Should there be a test checking what is returned and its shape?